src/main/java/nextstep/subway/line/domain/StationPosition.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.subway.line.domain;
+
+public enum StationPosition {
+    UP_STATION,
+    DOWN_STATION;
+
+    public boolean isUpStation() {
+        return this == UP_STATION;
+    }
+
+    public boolean isDownStation() {
+        return this == DOWN_STATION;
+    }
+
+    public boolean isOpposite(StationPosition stationPosition) {
+        return stationPosition != null & this != stationPosition;
+    }
```

Comment:
오타가 있는 것 같은데 맞을까요?? 🤔🤔

```suggestion
    public boolean isOpposite(StationPosition stationPosition) {
        return stationPosition != null && this != stationPosition;
    }
```

Reply:
헉.. 수정했습니다ㅎㅎ